symphony/bdk/core/auth/ext_app_authenticator.py

```diff
@@ -50,11 +51,7 @@ async def validate_jwt(self, jwt: str) -> dict:
         :return: the dictionary of jwt claims
         :raise AuthInitializationError: If the pod certificate or jwt are invalid.
         """
-        try:
-            pod_certificate = await self._get_pod_certificate()
-        except ApiException as exc:
-            raise AuthInitializationError(f"Unable to get the pod certificate: {exc.reason}") from exc
```

Comment:
Instead of raising an AuthInitializationError, I am raising the ApiException (same retry strategy as the authentication), we can see from the stack trace that this exception is originating from _get_pod_certificate